packages/models/tests/server/organizations_test.coffee

```diff
@@ -14,7 +14,19 @@ describe 'Organization attributes', ->
     organization.save
     expect(organization.description).to.eq('Description')
 
+
   it 'includes createdById', ->
     organization.set('createdById', 'fakeid')
     organization.save
     expect(organization.createdById).to.eq('fakeid')
+
+describe 'Organization truncated description', ->
```

Comment:
No, I just think you should combine this with the previous ones instead of having it by itself.